Core/src/org/sleuthkit/autopsy/timeline/ui/filtering/datamodel/SqlFilterState.java

```diff
@@ -74,7 +64,6 @@ public String getDisplayName() {
 
     @Override
     public String toString() {
-        activeProp.get();
```

Comment:
You can replace this with a call to something like getActiveProp() from AbstractFilterState.

Reply:
It's not needed.  That's why I removed it.